src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -479,14 +522,157 @@ void shouldDisplaySeriesPosition_withSeriesPositionPopulated_whenBookHasSeriesPo
     @Test
     void shouldNotDisplaySeriesPosition_whenBookDoesNotHaveSeriesPosition() {
         // given
-        bookForm = createBookForm(false);
+        bookForm = createBookForm(READ, false);
 
         bookForm.openForm();
 
         // then
         Assertions.assertFalse(bookForm.seriesPositionFormItem.isVisible());
     }
 
+    /**
+     * Tests whether a book is saved to the database when the save-event is called
+     */
+    @Test
+    void shouldSaveBookToDatabase() {
```

Comment:
Could you change the name here to be more descriptive such that the comment isn't needed (e.g. `bookSavedWhenSaveEventCalled`, `saveEventCalled_bookSavedToDatabase` or something better)?

Reply:
done